packages/view/backend/sirius-components-view-diagram/src/main/resources/model/diagram.ecore

```diff
@@ -16,6 +16,7 @@
     <eLiterals name="Circle" value="11" literal="Circle"/>
     <eLiterals name="FillCircle" value="12" literal="FillCircle"/>
     <eLiterals name="CrossedCircle" value="13" literal="CrossedCircle"/>
+    <eLiterals name="ClosedArrowWithVerticalBar" value="14"/>
```

Comment:
It is normal that there is no `literal=` for this value?
Not sure if there's an impact (maybe it defaults to the `name`?), but wondering about the difference with the others.

Reply:
No it's not, I don't know what happened here, I will fix that.